examples/geospatial/plot_delaunay.py

```diff
@@ -0,0 +1,54 @@
+"""
+=====
+Graphs from geographic points
+=====
+
+This example shows how to build a delaunay graph (plus its dual,
+the set of Voronoi polygons) from a set of points.
+For this, we will use the set of cholera cases at the Broad Street Pump, 
+recorded by John Snow in 1853. The methods shown here can also work 
+directly with polygonal data using their centroids as representative points. 
+"""
+
+from libpysal import weights, examples
+from libpysal.cg import voronoi_frames
+from contextily import add_basemap
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy
+import geopandas
+
+# read in example data online
```

Comment:
```suggestion
# read in example data in GeoPackage format
```
or similar

Reply:
I've added a little more explanation to each format's read.